README.md

```diff
@@ -121,6 +121,8 @@ ROS2 Distro | Branch | Build status | Documentation | Released packages
                 "kr150_2_macro.xacro",
                 "kr150r3100_2_macro.xacro",
                 "kr210l150_macro.xacro",
+                "kr210r3100_macro.xacro",
+                "lbr_iiwa_14_r820_macro.xacro",
```

Comment:
why is this "lbr_iiwa_14_r820_macro.xacro" here? it is a little bit confusing as the pr name is only about adding kuka210r3100